README.md

```diff
@@ -16,4 +16,9 @@ The Technical Requirements for this project can be found [here](https://backend.
 
 ### Evaluation
 
-Evaluation details can be found [here](https://backend.turing.io/module3/projects/rails_engine/evaluation)
\ No newline at end of file
+Evaluation details can be found [here](https://backend.turing.io/module3/projects/rails_engine/evaluation)
+
+
+### Models Schema Diagram
+
+![schema diagram](schema_diagram.png)
```

Comment:
Added diagram to the documentation with the idea of using it when I complete the final README for this project